data.json

```
@@ -611,6 +611,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/29686102/106919215-f0711a00-672f-11eb-88f6-8ab44f67b911.png",
     "issueNumber": 313
   },
+  {
+    "name": "Henok B Ademtew",
```

I copied their name from their GitHub bio, and I think it'd be better.

```suggestion
    "name": "Henok Biadglign Ademtew",
```

Or

```suggestion
    "name": "Henok B. Ademtew",
```

too.